src/pioneer-p3dx/p3dx_dpl/launch/dpl.launch

```diff
@@ -3,19 +3,19 @@
   <group ns="dpl">
     <!-- hal is foreign dpl member -->
     <include file="$(find p3dx_hal_vrep)/launch/hal_vrep.launch" />
-
+    <!-- <include file="$(find p3dx_dpl)/launch/robot_localization_ekf.launch" /> -->
```

Comment:
Why not? As soon as I find out how to fix it so it will work with mapping It will be part of the main code. For now it is easier to work with it when I am trying to debug robot_localization. It is pain to still write it and then delete it only because of every commit. This way it doesn't make any harm.